doc/source/user_guide/sparse.rst

```diff
@@ -288,6 +349,9 @@ row and columns coordinates of the matrix. Note that this will consume a signifi
 (relative to ``dense_index=False``) if the sparse matrix is large (and sparse) enough.
 
 .. ipython:: python
```

Comment:
is there a reason you are not changing this?

Reply:
Just laziness.